### PR TITLE
Fix toXML for CompleteMultipartUpload

### PR DESCRIPTION
**Pull request by EduardSergeev**
Fix `toXML` instance for `CompletedPart` which is used in `CompleteMultipartUpload` request.

### CODE REVIEW

File: amazonka-s3/gen/Network/AWS/S3/Types.hs [hunk 1]
```diff
@@ -2666,7 +2666,7 @@ instance FromXML CompletedPart where
         <*> x .@? "PartNumber"
 
 instance ToXML CompletedPart where
-    toXML CompletedPart{..} = nodes "CompletedPart"
+    toXML CompletedPart{..} =
```
Comment (charleso) [inline amazonka-s3/gen/Network/AWS/S3/Types.hs]: Add comment so people can see the only actual change from the overview. :)

**Comment by charleso** [comment]
You mind adding a note to the `CHANGES.md` file as well?

Also, is this fixed in the `develop` branch?

**Comment by charleso** [comment]
:+1:

**Comment by EduardSergeev** [comment]
@charleso I haven't tested `develop` but the code in it seems to be doing the right thing:
```haskell
instance ToXML CompletedMultipartUpload where
        toXML CompletedMultipartUpload'{..}
          = mconcat [toXML (toXMLList "Part" <$> _cmuParts)]
```

File: CHANGES.md [hunk 2]
```diff
@@ -15,3 +15,5 @@ List of changes on this fork:
              fixed `ListObject` pagination. Fixed on the `feature/ast` branch [#128](https://github.com/brendanhay/amazonka/issues/128)
 
  * `99.0.5`: fail on 3xx response codes
+
+ * `99.0.6`: fixed XML produced by `CompleteMultipartUpload` (in `ToXML` instance for `CompletedPart`).
```
Comment (EduardSergeev) [inline CHANGES.md]: Added changes.

**Comment by charleso** [comment]
Brilliant. Thanks @EduardSergeev :+1:

**Comment by nhibberd** [comment]
Looks good :+1: 
Good idea to split up the change and the version rename into seperate commits, easier to review based on commit that way

**Comment by EduardSergeev** [comment]
@nhibberd Makes sense. What's the process then: I create two commits then after review squash them into one before merge? Or is it ok to merge PR with two commits in this case?

**Comment by nhibberd** [comment]
In the past I've just merged with two commits

**Pull request merged**